### DISCUSSION TITLE
Migration from mobx to flutter_data

### DISCUSSION THREAD

**Discussion by mdmota** [flutterdata/flutter_data#146] [Q&A]
I have an App ready using mobx, get_it, shared_prefs.
Basically I have a model that is persisted locally and parts of it will be synced with api.
Some parts will change on demand and others will change whenever the app is opened.

Any difficulty migrating this type of project to flutter_data?

**Comment by frank06** [comment]
I have never used mobx so I wouldn't know the difficulty to migrate. What you describe is definitely possible.
